testing/sdk_tasks.py

```diff
@@ -67,12 +67,13 @@ def fn():
 
 
 def check_tasks_not_updated(service_name, prefix, old_task_ids):
+    # ensure new tasks are all launched and complete, either via deploy or recovery
     sdk_plan.get_deployment_plan(service_name)
+    sdk_plan.get_recovery_plan(service_name)
```

Comment:
Just getting the plans no longer guarantees they're complete.  You'll have to use the new `sdk_plan` methods that check for plan completeness.